engine/src/main/java/org/terasology/engine/world/chunks/RenderableChunk.java

```diff
@@ -2,37 +2,31 @@
 // SPDX-License-Identifier: Apache-2.0
 package org.terasology.engine.world.chunks;
 
+import org.joml.Vector3f;
 import org.terasology.joml.geom.AABBfc;
 import org.terasology.module.sandbox.API;
 import org.terasology.engine.rendering.primitives.ChunkMesh;
 
 /**
+ * Anything that acts like a chunk for rendering purposes
  */
 @API
-public interface RenderableChunk extends LitChunk {
+public interface RenderableChunk {
 
-    boolean isDirty();
-
-    void setDirty(boolean dirty);
+    Vector3f getRenderPosition();
```

Comment:
The components of the mesh have coordinates defined relative to this origin. For a normal chunk, it's 0.5 blocks inside one of the corners, but it can be arbitrary.